server/activate_hooks.go

```diff
@@ -82,17 +82,6 @@ func (p *Plugin) OnDeactivate() error {
 		if err := p.postPluginMessage(team.Id, msg); err != nil {
 			return errors.Wrap(err, "failed to post OnDeactivate message")
 		}
-
-		if err := p.API.UnregisterCommand(team.Id, CommandTriggerPlugin); err != nil {
```

Comment:
Why are the UnregisterCommand's removed?  Are they no longer needed?

Reply:
Yes, this is not longer needed. There was a bug, but this is fixed since a few month. See https://github.com/mattermost/mattermost-server/pull/10086

Reply:
Awesome!  I will make the changes in my own plugins too.

Reply:
I also just fixed this for Matterpoll :wink: https://github.com/matterpoll/matterpoll/pull/164